src/attr/__init__.pyi

```diff
@@ -37,6 +37,10 @@ class Attribute(Generic[_T]):
     type: Optional[Type[_T]]
 
 
+# FIXME: if no type arg or annotation is provided when using `attr` it will result in an error:
+# error: Need type annotation for variable
```

Comment:
I don't think this needs to be fixed. From a typing perspective you want the coder to be explicit about the type if it's not specified, not just default to `Any`.  (In fact in my plugin I did make it do that)

Reply:
I read your post on the mypy issue and I can definitely see things from your perspective.  I'll adjust the comment to reflect that this is a potential topic for debate.